### ISSUE TITLE
Purpose of selected items span in BasePicker?

### ISSUE THREAD

**Issue by Ridlsan** [OfficeDev/office-ui-fabric-react#6696]
![image](https://user-images.githubusercontent.com/13232749/46956783-d3b40280-d09e-11e8-899d-6b9138e5c110.png)
What is the purpose of this span in BasePicker ? If this span exist, input (red color) is droping next row, but there is still space for it upper (green color). 
In previous versions there were no span, so i made my tag-items and input flex.
Why has u added it? Can u remove it and make it like it was before? 
![image](https://user-images.githubusercontent.com/13232749/46956802-e3cbe200-d09e-11e8-9841-8a8d94825d3d.png)

**Comment by dzearing** [comment]
@joschect i'm guessing this is to contain the items, maybe was a result of pre-React 16 lack of ability to return an array of children or something?

**Comment by Ridlsan** [comment]
@dzearing @joschect no, it seems like was added in last [change](https://github.com/OfficeDev/office-ui-fabric-react/commit/f032d31e401e9d7ef300224569129db698447632#diff-c88a174b6c194e2d439942d05cb8fb6c)

**Comment by joschect** [comment]
@Ridlsan and @dzearing the purpose is for accessibility. It lets the autofill, which is what usually gets focused, have the describedby attribute point just to the selected items

**Comment by Ridlsan** [comment]
@dzearing @joschect bug in bot ?)
